web/styles/left_sidebar.css

```diff
@@ -137,22 +137,70 @@ li.show-more-topics {
         }
     }
 
-    .subscription_block .unread_count {
-        margin-right: 15px;
-    }
-
     .subscription_block {
         margin-right: 25px;
         margin-left: $far_left_gutter_size;
         display: flex;
         align-items: center;
         white-space: nowrap;
 
+        & .unread_count {
+            margin-right: 15px;
+        }
+
+        & .masked_unread_count {
+            margin-right: 19px;
+        }
```

Comment:
Can you add some comments documenting these magic numbers?

Reply:
So basically we have horizontal padding of 4px in addition to 15px margin for `unread_counts` span applied in `app_components.css`. Therefore to keep `masked_unread_count` span horizontally centered with `unread_counts` span we will need to add those extra 4px of padding applied to `unread_counts` and hence a total margin of `15px + 4px = 19px`.